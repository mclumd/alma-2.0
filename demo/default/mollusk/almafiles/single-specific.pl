rel(naked_nautilus, steve).
if(now(0), obs(quote(rel(nautilus, steve)))).
if(now(3), obs(quote(rel(cephalopod, steve)))).
if(now(5), obs(quote(rel(mollusk, steve)))).
