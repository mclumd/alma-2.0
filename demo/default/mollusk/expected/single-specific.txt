~rel(shell_bearer, steve)
rel(naked_nautilus, steve)
rel(nautilus, steve)
rel(cephalopod, steve)
rel(mollusk, steve)
abnormal(steve, mollusk, "~rel(shell_bearer, steve)")
abnormal(steve, nautilus, "~rel(shell_bearer, steve)")
